app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,30 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :authorize_user!, only: [:destroy]
+
+  expose_decorated(:comment)
+
+  def create
+    if comment.save
+      redirect_to article_path(comment.article), notice: 'Comment created'
+    else
+      redirect_to article_path(comment.article), alert: 'Comment error'
```

Comment:
pls try to use this gem: https://github.com/plataformatec/responders

it would be cool if you can replace that redirects with 'respond_with' method.